crawler.py

```diff
@@ -163,19 +169,19 @@ def run(website, driver):
         print('Continuing...')
 
     saved_sequence = []
-    try:
-        for i in range(0, 20):
+    for i in range(0, 20):
+        try:
             elem_attributes = do_something(driver)
             if elem_attributes is None:
                 print('Can\'t find any element to interact with on {}'.format(website))
                 break
             saved_sequence.append(elem_attributes)
 
             print('  - Using {}'.format(elem_attributes))
-    except TimeoutException as e:
-        # Ignore timeouts, as they are too frequent.
-        traceback.print_exc()
-        print('Continuing...')
+        except (TimeoutException, ElementNotInteractableException, StaleElementReferenceException, WebDriverException):
```

Comment:
Can you post examples of these exceptions? Maybe it's fine to ignore `ElementNotInteractableException` and `StaleElementReferenceException`, but maybe we shouldn't ignore `WebDriverException`.

Reply:
`StaleElementReferenceException` - selenium.common.exceptions.StaleElementReferenceException: Message: The element reference of a class="select-menu-item js-navigation-item js-navigation-open " href="/mozilla/bedrock/blob/bug1150328-wom-landing-page/bedrock/mozorg/templates/mozorg/home/home-en.html" is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed
`ElementNotInteractableException` - selenium.common.exceptions.ElementNotInteractableException: Message: Element \<a href="/users/496127/pwan"\> could not be scrolled into view
`WebDriverException` - selenium.common.exceptions.WebDriverException: Message: Element <button class="ProfileTweet-actionButton js-actionButton js-actionFavorite" type="button"> is not clickable at point (637,9.350006103515625) because another element <div class="container"> obscures it

Reply:
Should I remove `WebDriverException`? It is quite frequent and interrupts crawling. Can you suggest another workaround to fix this?

Reply:
Let's keep it, I didn't know what kinds of exceptions would be hidden behind WebDriverException, but the one you mentioned seems not too bad.